src/llmq/dkgsession.cpp

```diff
@@ -372,13 +372,13 @@ void CDKGSession::VerifyPendingContributions()
         }
         memberIndexes.emplace_back(idx);
         vvecs.emplace_back(receivedVvecs[idx]);
-        skContributions.emplace_back(receivedSkContributions[idx]);
+        skContributions2.emplace_back(receivedSkContributions[idx]);
         // Write here to definitely store one contribution for each member no matter if
         // our share is valid or not, could be that others are still correct
         dkgManager.WriteEncryptedContributions(params.type, m_quorum_base_block_index, m->dmn->proTxHash, *vecEncryptedContributions[idx]);
     }
 
-    auto result = blsWorker.VerifyContributionShares(myId, vvecs, skContributions);
+    auto result = blsWorker.VerifyContributionShares(myId, vvecs, skContributions2);
```

Comment:
Missing s/`skContributions[i]`/`skContributions2[i]`/ at line 398 below. Also, smth2 is not very informative ("why 2?"). Let's maybe rename the class member instead e.g. `m_sk_contributions`?
>- Variable and namespace names are all lowercase, and may use _ to separate words (snake_case).
>    -  Class member variables have a m_ prefix.

https://github.com/dashpay/dash/blob/master/doc/developer-notes.md#coding-style-c

Reply:
That would be a good solution :) I'll do that later (feel free to push a commit for it if you like)

Reply:
Done :)